torch/csrc/jit/script/compiler.cpp

```diff
@@ -727,6 +728,30 @@ static Value* emitBuiltinNode(
   return packOutputs(graph, n->outputs());
 }
 
+using OperatorMap = std::pair<Symbol, std::vector<std::shared_ptr<Operator>>>;
+
+std::multimap<unsigned, OperatorMap> findOperatorsWithSimilarName(Symbol name) {
+  std::string user_function_name(name.toQualString());
+
+  // currently only using the name of the op, could be improved to check
+  // similarity of arguments
+  using ranking_pair = std::pair<unsigned, OperatorMap>;
+  std::multimap<unsigned, OperatorMap> rankings;
+
+  auto all_operators = getAllOperators();
+  const size_t MAX_EDIT_DIST = 2u;
+  auto best_dist = MAX_EDIT_DIST;
+  for (auto op: all_operators) {
+    std::string builtin_op_name = op.first.toQualString();
```

Comment:
Copying every operator name is unnecessary. if `computeEditDistance` took an ArrayRef or const chart * you can avoid it entirely

Reply:
The copy is made in the line I indicated, when you invoke the copy constructor of `std::string`.

`const auto` would have copied the const char *, which is just a pointer that you could then pass to computeEditDistance without a copy.